src/compiler/types.ts

```diff
@@ -2719,6 +2719,7 @@ namespace ts {
         inlineSources?: boolean;
         isolatedModules?: boolean;
         jsx?: JsxEmit;
+        jsxFactory?: string;

```

Comment:
Instead of parsing the `jsxFactory` to an `EntityName` multiple times through-out the program, would it be better to have this be `EntityName` and parse it once up-front?